### PR TITLE
contrib: add validation for kubectl

### PR DESCRIPTION
**Pull request by dougsland**
kubectl is required during the build, we must
validate if the bin exists, if not, download it.

Signed-off-by: Douglas Schilling Landgraf <dougsland@redhat.com>

### CODE REVIEW

**Comment by dougsland** [comment]
/assign @jcaamano

Not every system works with `devmaster` (which install kubectl in fedora). Tested against ubuntu `kind.sh` and failed with missing kubectl. (Users have asked regarding ubuntu)

File: contrib/kind.sh [hunk 1]
```diff
@@ -293,6 +336,16 @@ install_j2_renderer() {
 }
 
 check_dependencies() {
+  if ! command_exists curl ; then
+    echo "Dependency not met: Command not found 'curl'"
+    exit 1
+  fi
+
+  if ! command_exists kubectl ; then
+    echo "Dependency not met: Command not found 'kubectl'"
```
Comment (jcaamano) [inline contrib/kind.sh:345]: Could you log that you are installing it, similar to how it is done with j2 below?

Comment (dougsland) [reply contrib/kind.sh:351]: @jcaamano done, thanks for the review.

File: contrib/kind.sh [hunk 2]
```diff
@@ -293,6 +342,17 @@ install_j2_renderer() {
...
+    echo "Downloading kubectl..'"
```
Comment (jcaamano) [inline contrib/kind.sh:352]: Sorry for the super nit but I would remove 
```
echo "Dependency not met: Command not found 'kubectl'"
```
and replace
```
echo "Downloading kubectl..'"
```
with
```
echo "'kubectl' not found, installing"
```
for commonality with what is printed for j2 below.

Comment (jcaamano) [inline contrib/kind.sh:352]: Sorry for the super nit but I would remove 
```
echo "Dependency not met: Command not found 'kubectl'"
```
and replace
```
echo "Downloading kubectl..'"
```
with
```
echo "'kubectl' not found, installing"
```
for commonality with what is printed for j2 below.

**Comment by trozet** [comment]
@dougsland can you respond to the comments here?

**Comment by dougsland** [comment]
@trozet just got back from parental leave. :)
@jcaamano Are you able to do cherry pick?  Please let me know if you want me a rebase or a fresh PR.

**Pull request merged**